files/bin/buildlibs.sh

```diff
@@ -139,6 +139,15 @@ function ensure_musl {
   if [[ ${OS} = "macosx" ]] ; then
       return
   fi
+  if [[ ! -f ${MUSL_GCC} ]] ; then
+      # if musl-gcc is already installed, use it
+      existing_musl=$(which musl-gcc 2> /dev/null)
+      if [[ -n "${existing_musl}" ]]; then
```

Comment:
if `musl-gcc` is already installed, we dont need to compile musl from scratch which requires more system deps like `make`. for example avoids errors like:

```
#14 15.44 /tmp/nimble_9/githubcom_crashappseccon4mgit_#ms/libs/files/bin/buildlibs.sh: line 156: make: command not found
```